infra/build/functions/update_build_status.py

```diff
@@ -39,6 +41,9 @@
 
 FUZZING_STATUS_FILENAME = 'status.json'
 COVERAGE_STATUS_FILENAME = 'status-coverage.json'
+PROJECTS_DIR = os.path.normpath(
```

Comment:
This won't work unfortunately, because this is not run as part of an OSS-Fuzz checkout. 

Instead, what we can do to fix this here is to do a datastore query for the Project entity:  https://github.com/google/oss-fuzz/blob/6ca344a44647fe3087d98e0f8c2a545f8285cca5/infra/build/functions/project_sync.py#L132

If it does not exist, then don't upload.

These entities are already being deleted when a project is deleted, but not when it's disabled: https://github.com/google/oss-fuzz/blob/6ca344a44647fe3087d98e0f8c2a545f8285cca5/infra/build/functions/project_sync.py#L117

Probably need to change something around here to check for the disabled attribute: 
https://github.com/google/oss-fuzz/blob/6ca344a44647fe3087d98e0f8c2a545f8285cca5/infra/build/functions/project_sync.py#L202